source/fundamentals/aggregation/aggregation-expression-operations.txt

```diff
@@ -25,8 +25,109 @@ more naturally readable.
    :manual:`Aggregation Pipeline Operators </reference/operator/aggregation/>` 
    page in the Server manual.
 
+Sample Data
+-----------
+
+To run the examples in this guide, insert documents describing locations
+and temperature readings into the ``db.tempReadings`` collection with the following
+snippet:
+
+.. code-block:: java
+   :copyable: true
+
+   MongoDatabase database = client.getDatabase("db");
+   MongoCollection<Document> myColl = database.getCollection("tempReadings");
+   
+   List<Document> docs = Arrays.asList(
+           new Document().append("location", "New York, NY").append("temp", 54),
+           new Document().append("location", "Duluth, MN").append("temp", 4),
+           new Document().append("location", "Riverside, CA").append("temp", 101),
+           new Document().append("location", "Tucson, AZ").append("temp", 92));
+
+   InsertManyResult result = myColl.insertMany(docs);
+
 Operations
 ----------
 
+Boolean Expressions
+~~~~~~~~~~~~~~~~~~~
+
+Boolean expressions evaluate arguments as booleans and return ``true`` or
+``false``. These expressions evaluate ``false``, ``null``, ``0``, and
+``undefined`` values as ``false``, and all other values as ``true``.
+
+The following table describes the boolean aggregation expression methods
+available in the {+driver-short+} and the equivalent expressions in the
+Query API:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 40 60
+
+   * - Java Aggregation Expression
+     - Query API Expression
+
+   * - ``and()``
+     - :manual:`$and </reference/operator/aggregation/and/>`
+       
+   * - ``not()``
+     - :manual:`$not </reference/operator/aggregation/not/>`
+
+   * - ``or()``
+     - :manual:`$or </reference/operator/aggregation/or/>`
```

Comment:
throughout our documentation, we refer to methods with a `()` marker, irrespective of the # of parameters they take. This format distinguishes methods from class references, etc. We think it is confusing for users to see placeholders such as `a` and `b`.

Reply:
Fair enough.

I think users must be ok with placeholders to some extent, since they are used prominently in the [expressions docs](https://www.mongodb.com/docs/manual/reference/operator/aggregation/or/), but I can see how they might also be too much here.